src/test/java/no/unit/nva/database/DatabaseServiceTest.java

```diff
@@ -46,7 +48,7 @@ public void init() {
     @Test
     public void databaseServiceHasAMethodForInsertingAUser()
         throws InvalidEntryInternalException, ConflictException, InvalidInputException {
-        UserDto user = createSampleUserWithoutInstitutionOrRoles(SOME_USERNAME);
+        UserDto user = createSampleUserWithoutInstitutionOrRoles(SOME_USERNAME, SOME_GIVEN_NAME, SOME_FAMILY_NAME);
```

Comment:
I suggest that we remove the parameters and we use the constants directly, since we do not use any other values for these parameters